requirements/tensorflow/env_tensorflow.yml

```diff
@@ -0,0 +1,25 @@
+name: deepchem
+channels:
+  - omnia
+  - conda-forge
+  - defaults
+dependencies:
+  - openmm
+  - pdbfixer
+  - rdkit
+  - mdtraj
+  - numpy==1.19.*
+  - pip==20.2.*
```

Comment:
Can these be simplified to only include dependencies that are not in the `common` env?

Reply:
Just one problem for now, Tensorflow uses numpy==1.19.*, but other environments are compatible with numpy==1.21.0 as well.